specs/core/0_beacon-chain.md

```diff
@@ -750,22 +856,25 @@ def get_attestation_participants(state: State,
     """
     Returns the participant indices at for the ``attestation_data`` and ``participation_bitfield``.
     """
-    sncs_for_slot = get_shard_and_committees_for_slot(state, attestation_data.slot)
-    snc = [x for x in sncs_for_slot if x.shard == attestation_data.shard][0]
-    assert len(participation_bitfield) == ceil_div8(len(snc.committee))
-    participants = []
-    for i, validator_index in enumerate(snc.committee):
-        bit = (participation_bitfield[i//8] >> (7 - (i % 8))) % 2
-        if bit == 1:
-            participants.append(validator_index)
-    return participants
+
+    # Find the relevant committee
+    shard_committees = get_shard_committees_at_slot(state, attestation_data.slot)
+    shard_committee = [x for x in shard_committees if x.shard == attestation_data.shard][0]
+    assert len(participation_bitfield) == ceil_div8(len(shard_committee.committee))
+
+    # Find the attesters in the committee
+    attesters = []
```

Comment:
IMO I think it's fine to name it `participants` here. "Attester" is their job/role, but only the participated ones will be added to this list.

Reply:
Ok 👍